### ISSUE TITLE
SA1625 (copy and paste) and `<see>` like element documentations.

### ISSUE THREAD

**Issue by foriequal0** [DotNetAnalyzers/StyleCopAnalyzers#3154]
I sometimes write miscellaneous comments outside `<summary>` or `<remarks>`.
Also, I link items with `<see cref="SomeClass"/>` a lot.

These two combinations makes StyleCop warns the following comments
```csharp
/// You should decorate classes with <see cref="SomeAttribute"/> when they derive this class.
/// Without <see cref="SomeAttribute"/>, it'll fail at runtime.
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~
warning SA1625: Element documentation should not be copied and pasted
```

Following combinations are warned:
* `<see cref="SomeAttribute"/>` and `<see cref="SomeAttribute"/>`
* `<see cref="SomeAttribute">this</see>` and `<see cref="SomeAttribute">this</see>`

But these are not:
* `<see cref="SomeAttribute"/>` and `<see cref="SomeAttribute">this</see>`

I find that SA1625 is useful to catch copy-and-paste mistakes, but I think this is a false positive case. I think they should be applied to limited elements (e.g. `<param>`, `<exception>`).

**Comment by sharwell** [comment]
> I sometimes write miscellaneous comments outside `<summary>` or `<remarks>`.

💭 This is not a valid documentation comment, so it's hard to say what kind of behavior would be "right" or "wrong" in such a scenario.

**Comment by foriequal0** [comment]
I understand that it is not a recommended way to write documentation comments. But I think it is not a completely invalid format according to this [document](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments)

I generate OpenAPI documentation with Swashbuckle. It converts `<summary>` to `Title`, and `<remarks>` to `Description`. They are exported publicly. But I also want to comment on miscellaneous things privately.

I will use plain `//` comment without the aid of IDE, or wrapping them in `<para>`.

Thanks.

**Comment by sharwell** [comment]
💡 I've also seen people wrap them in `<devdoc>`.
